dkc/core/models/file.py

```diff
@@ -63,6 +61,6 @@ def clean(self) -> None:
 
 
 @receiver(models.signals.pre_save, sender=File)
-def _file_pre_save(sender, instance, *args, **kwargs):
+def _folder_pre_save(sender: Type[File], instance: File, **kwargs):
```

Comment:
@brianhelba was this function name change intentional?

Reply:
No, got stuck in an ugly rebase. I'll fix.

Thanks for the catch.